<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AS400InputWidget</class>
 <widget class="QWidget" name="AS400InputWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="0">
    <widget class="QPushButton" name="as400CommitSettings">
     <property name="text">
      <string>Commit Settings</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="as400PasswordLabel">
     <property name="text">
      <string>AS400 Password</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="as400DriverLineEdit">
     <property name="text">
      <string>iSeries Access ODBC Driver</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLineEdit" name="as400SystemLineEdit">
     <property name="text">
      <string>Insert system IP or hostname here</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="as400DriverLabel">
     <property name="text">
      <string>AS400 ODBC Driver</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLineEdit" name="as400PasswordLineEdit">
     <property name="text">
      <string>password</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLineEdit" name="as400UsernameLineEdit">
     <property name="text">
      <string>Insert username here</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="as400UsernameLabel">
     <property name="text">
      <string>AS400 Username</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="as400SystemLabel">
     <property name="text">
      <string>AS400 Hostname or IP</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="as400LoadSettingsButton">
     <property name="text">
      <string>Load Existing Settings</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>as400LoadSettingsButton</tabstop>
  <tabstop>as400DriverLineEdit</tabstop>
  <tabstop>as400SystemLineEdit</tabstop>
  <tabstop>as400UsernameLineEdit</tabstop>
  <tabstop>as400PasswordLineEdit</tabstop>
  <tabstop>as400CommitSettings</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
